static/app/views/app/index.tsx

```diff
@@ -170,6 +172,18 @@ function App({children, params}: Props) {
       );
     }
 
+    if (partnershipAgreementPrompt) {
+      return (
+        <Suspense fallback={null}>
+          <PartnershipAgreement
+            partnerDisplayName={partnershipAgreementPrompt.partnerDisplayName}
+            agreements={partnershipAgreementPrompt.agreements}
+            onSubmitSuccess={() => ConfigStore.set('partnershipAgreementPrompt', null)
```

Comment:
nit: Maybe just `onSuccess`?

Reply:
It's a form and the property on form is called `onSubmitSuccess` so keeping it.